index.bs

```diff
@@ -729,23 +729,23 @@ The {{Element/requestFullscreen|Element.requestFullscreen()}} method steps are u
 ## Permission API Integration ## {#api-permission-api-integration}
```

Comment:
This section should document the rename, and plans to phase out support for the legacy string.

Reply:
Added a note about the rename.